src/Dilex.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+
+namespace Clearbooks\Dilex;
+
+use Silex\Application;
+
+class Dilex extends Application implements RouteContainer, MiddlewareContainer
+{
+    /**
+     * @param string $pattern
+     * @param Endpoint $to
```

Comment:
These type hints are wrong, should be string. Or rather a ::class of type Endpoint